pkg/acceptance/bettertestspoc/assert/objectassert/view_snowflake_ext.go

```diff
@@ -28,3 +32,71 @@ func (v *ViewAssert) HasNonEmptyText() *ViewAssert {
 	})
 	return v
 }
+
+func (v *ViewAssert) HasNoRowAccessPolicyReferences() *ViewAssert {
+	return v.hasNoPolicyReference(sdk.PolicyKindRowAccessPolicy)
+}
+
+func (v *ViewAssert) HasNoAggregationPolicyReferences() *ViewAssert {
+	return v.hasNoPolicyReference(sdk.PolicyKindAggregationPolicy)
+}
+
+func (v *ViewAssert) HasNoMaskingPolicyReferences() *ViewAssert {
+	return v.hasNoPolicyReference(sdk.PolicyKindMaskingPolicy)
+}
+
+func (v *ViewAssert) HasNoProjectionPolicyReferences() *ViewAssert {
+	return v.hasNoPolicyReference(sdk.PolicyKindProjectionPolicy)
+}
+
+func (v *ViewAssert) hasNoPolicyReference(kind sdk.PolicyKind) *ViewAssert {
+	v.AddAssertion(func(t *testing.T, o *sdk.View) error {
+		t.Helper()
+		refs, err := acc.TestClient().PolicyReferences.GetPolicyReferences(t, o.ID(), sdk.ObjectTypeView)
```

Comment:
we should not add here acc.TestClient() because they can be invoked both from acc and integration tests levels, please for now parameterize invocation with a proper test client

Reply:
Don